.yarnrc.yml

```
@@ -1 +1 @@
-nodeLinker: node-modules
+nodeLinker: node-modules
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)